bower.json

```diff
@@ -47,8 +47,6 @@
     "patternfly-timeline": "~1.0.3",
     "phantomjs-polyfill": "~0.0.2",
     "qs": "~0.3.10",
-    "rx-angular": "rx.angular#~1.1.3",
```

Comment:
Looks like we will need to begin to remove rx-angular from https://github.com/ManageIQ/ui-components as well

@karelhala ^^